community/modules/compute/schedmd-slurm-gcp-v5-partition/main.tf

```diff
@@ -68,6 +71,9 @@ locals {
   slurm_cluster_name = var.slurm_cluster_name != null ? var.slurm_cluster_name : local.tmp_cluster_name
 }
 
+data "google_compute_default_service_account" "default" {
```

Comment:
nit: It is not clear to me why this change was needed now. Maybe just add a note in the description with the "why".

Feel free to merge without sending back to me.